roles/kubevirt_web_ui/tasks/main.yml

```diff
@@ -2,7 +2,7 @@
 # do any asserts here
 
 - include_tasks: install.yml
-  when: kubevirt_web_ui_install | default(true) | bool
+  when: apb_action | default("provision") | string == "provision"
```

Comment:
if you rename install.yaml and remove.yml to 'provision' and 'deprovision' you can do:
```yaml
- include_tasks: {{ apb_action}}.yml
```